target/linux/ramips/image/mt7621.mk

```diff
@@ -753,6 +753,17 @@ define Device/iptime_a8004t
 endef
 TARGET_DEVICES += iptime_a8004t
 
+define Device/iptime_a3004nd
+  $(Device/dsa-migration)
+  $(Device/uimage-lzma-loader)
+  IMAGE_SIZE := 16128k
+  UIMAGE_NAME := a3004nd
+  DEVICE_VENDOR := ipTIME
+  DEVICE_MODEL := A3004NS-dual
+  DEVICE_PACKAGES := kmod-usb3 kmod-mt76x2
```

Comment:
Probably you'll also need `kmod-usb-ledtrig-usbport`.

Reply:
Hello, I added the package on the DEVICE_PACKAGES section. Thanks.